src/qibosoq/qick_programs.py

```diff
@@ -16,7 +16,12 @@
 NS_TO_US = 1e-3
 
 
-class FluxProgram:
+class FluxProgram:  # TODO include and test
+    @staticmethod
+    # TODO use
+    def from_volt_to_gain(volt: float, p0: int = 329, p1: int = 25634):
+        return int(p0 + volt * p1)
```

Comment:
Static methods are just a way to put something in a given namespace, since there is no reference to `self` or `cls`.

Consider if this does really belong to this namespace, or it can just be lifted top-level.